src/Smart.Design.Razor/TagHelpers/Alerts/SmartAlertStackTagHelper.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNetCore.Mvc.TagHelpers;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using Smart.Design.Razor.Enums;
+using Smart.Design.Razor.TagHelpers.Constants;
+using Smart.Design.Razor.TagHelpers.Extensions;
+using Smart.Design.Razor.TagHelpers.Html;
+
+namespace Smart.Design.Razor.TagHelpers.Alerts
+{
+    [HtmlTargetElement(TagNames.SmartAlertStack, TagStructure = TagStructure.NormalOrSelfClosing)]
```

Comment:
General question for the team: do we really need to prefix every name with "Smart" ?

Reply:
The most common way of naming namespaces is the following:
[CompanyName].[Product/Product].[Modules].[Etc].

Reply:
Ah you were most likely talking about the constants, variables names ?

I chose to name elements and components with the prefix Smart to avoid conflict with potential third party library, we won't be facing this problem, however, user of ours lib might.

On another hand, it shows that what you using are actually Smart's stuff.

Downsize is word are longer.

Reply:
I was talking about things like `TagNames.SmartAlertStack`.